input/pagecontent/index.md

```diff
@@ -17,6 +17,25 @@ The AU eRequesting project includes:
 
 {% include dependency-table.xhtml %}
 
+### AU eRequesting Actors
+
+The following actors are part of AU eRequesting:
+
+**AU eRequesting Placer**
+
+A system is responsible for initiating and creating patient diagnostic service requests. This system connects to enable requesters to provide the necessary information to create a request, ensuring that all required data is accurately collected and provided to service providers with sufficient information for them to fulfil the request.  
+
+
+**AU eRequesting Filler**
+
+A system responsible for finding and retrieving diagnostic service requests so that service providers can fulfil them It ensures that service providers have the necessary details to fulfil the requests and manages the requests through the fulfilment process.
+
+
+**AU eRequesting Patient Access**
+
+A system is the digital interface that allows patients or their representatives to view and manage their requests for diagnostic services. It provides access to request details, enabling patients to choose service providers and track the status of their requests.
```

Comment:
Get rid of the second "their" so it says "manage requests for diagnostic services". Replace "details" with "information". 
Remove the third "their" so it says "track the status of requests".

Reply:
Addressed via PR #39